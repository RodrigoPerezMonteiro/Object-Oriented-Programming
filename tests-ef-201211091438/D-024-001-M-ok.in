6
3
Tomate palha
salada mista
70
batata palha
30
5
Tomate palha
0
0
