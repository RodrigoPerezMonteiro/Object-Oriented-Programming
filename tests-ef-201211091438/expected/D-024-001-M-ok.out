Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Nome a dar ao alimento: Identificador do alimento (nome): Percentagem do alimento (inteiro entre 0 e 100): Identificador do alimento (nome): Percentagem do alimento (inteiro entre 0 e 100): Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Identificador do alimento (nome): VEG|Tomate palha
VEG|salada mista|70
VEG|salada mista
VEGETABLE|alface|50
VEGETABLE|tomate|45
VEGETABLE|azeite|5
VEG|batata palha|30
VEG|batata palha
VEGETABLE|batata|78
VEGETABLE|azeite|22
Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 
